docs/architecture/assets.md

```diff
@@ -1,58 +1,81 @@
-In Miden, users can create and trade arbitrary fungible and non-fungible assets.
+# Asset
 
-We differentiate between native and non-native assets in Miden. Native assets follow the Miden asset model. Non-native assets are all other data structures of value that can be exchanged.
+> Fungible and Non-fungible assets in the Miden protocol.
 
-Native assets in Polygon Miden have four goals:
+## What is the purpose of an asset?
 
-* Asset exchange should be parallelizable.
-* Asset ownership should be self-sovereign.
-* Asset usage should be censorship resistant.
-* Fees can be paid using any asset.
+In Miden, assets serve as the primary means of expressing and transferring value between [accounts](accounts.md) through [notes](notes.md). They are designed with four key principles in mind:
 
-All native assets in Miden are stored directly in accounts, like Ether in Ethereum. Miden does not track asset ownership using global hashmaps, e.g., ERC20 contracts. Local asset storage in accounts provides privacy and the ability for client-side proofs. That is because ownership changes are reflected only on an account and not in an ERC20 account (global hashmap). Thus, these changes can happen in parallel. Additionally, asset exchange is censorship resistant at this level because there is no global contract the transfer must pass through. Finally, users can pay fees in any asset.
+1. **Parallelizable exchange:**  
+    By managing ownership and transfers directly at the account level instead of relying on global structures like ERC20 contracts, accounts can exchange assets simultaneously, boosting scalability and efficiency.
 
-## Native assets
+2. **Self-sovereign ownership:**  
+   Assets are stored in the accounts directly. This ensures that users retain complete control over their assets.
 
-Native assets are data structures that follow the Miden asset model (encoding, issuance, storing). All native assets are encoded using a single `word` (4 field elements). The asset encodes both the ID of the issuing account and the asset details. 
+3. **Censorship resistance:**  
+   Users can transact freely and anonymously with no single contract or entity controlling asset transfers. This reduces the risk of censored transactions, resulting in a more open and resilient system.
 
-Having the issuer's ID encoded in the asset makes determining the type of an asset, inside and outside Miden VM, cost-efficient. And, representing the asset in a `word` means the representation is a commitment to the asset data itself. That is particularly interesting for non-fungible assets.
+4. **Flexible fee payment:**  
+   Unlike protocols that require a specific base asset for fees, Miden allows users to pay fees in any supported asset. This flexibility simplifies the user experience.
 
-### Issuance
+## What is an asset?
+
+An asset in Miden is a unit of value that can be transferred from one [account](accounts.md) to another using [notes](notes.md).
```

Comment:
As mentioned further above, this is basically the definition of an Asset. So this should go directly to the top where other sections (like Notes, Accounts) contain their definition.